services/graphite/service.go

```diff
@@ -21,6 +23,32 @@ const (
 	leaderWaitTimeout = 30 * time.Second
 )
 
+// expvar stats maintained by the Graphite package.
+var epOnce sync.Once
```

Comment:
These `ep` and `EP_` prefixes don't read very well.  Not really clear what `EP` means at first glance.  Maybe use `stat` and `STAT` or drop them altogether?